packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1312,6 +1371,14 @@ export function getFullColorString(color: IColor): string;
 // @public (undocumented)
 export const getIconContent: (iconName?: string | undefined) => IIconContent | null;
 
+// Warning: (ae-internal-missing-underscore) The name "getItemClassNames" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal @deprecated (undocumented)
+export const getItemClassNames: (theme: ITheme, disabled: boolean, expanded: boolean, checked: boolean, isAnchorLink: boolean, knownIcon: boolean, itemClassName?: string | undefined, dividerClassName?: string | undefined, iconClassName?: string | undefined, subMenuClassName?: string | undefined, primaryDisabled?: boolean | undefined, className?: string | undefined) => IContextualMenuItemStyles;
+
+// @public
+export const getItemStyles: (props: IContextualMenuItemStyleProps) => IContextualMenuItemStyles;
```

Comment:
This probably ought to be exported under a different name like `getContextualMenuItemStyles` (`getItemStyles` is too generic for an export)

Reply:
Fixed.